docs/api/browser-view.md

```diff
@@ -41,7 +41,10 @@ Objects created with `new BrowserView` have the following properties:
 
 #### `view.webContents` _Experimental_
 
-A [`WebContents`](web-contents.md) object owned by this view.
+* `webContents` [WebContents](web-contents.md)
```

Comment:
@Juice10 bullet points aren't stylistically expected in our parsing code - this should be removed. Simply adding what you already did below:

> Returns `WebContents | null` - The `WebContents` owned by this view
or `null` if the contents are [destroyed](web-contents.md#event-destroyed).

Fixes the issue. You can verify this by looking at the `Archaeologist` check in CI.

Reply:
Replacing `Returns` with `A` fixed the issue!

I'm getting another issue now but it's no longer a docs parsing issue, but a typing issue so this is progress! Thanks for the help @codebytere